scripts_ZA/postProcessPlotterOutput.sh

```diff
@@ -60,24 +60,22 @@ for d in ${directories[*]}; do
     #fi
 
     # take envelopes for scale systematics
-    #if [[ ${file_content} =~ scaleUncorr || ${file_content} =~ dyScaleUncorr ]]; then
-    #    echo "Creating scale systematics..."
-    #    parallel -j 5 createScaleSystematics.py -s scaleUncorr dyScaleUncorr -- ::: *.root
-    #else
-    #    echo "No scale systematics found!"
-    #fi
-
-    #parallel -j 4 ../../../../scripts/fixJER.py -i ::: *.root
+    if [[ ${file_content} =~ scaleUncorr ]]; then
+        echo "Creating scale systematics..."
+        parallel -j 5 createScaleSystematics.py -s scaleUncorr -- ::: *.root
+    else
+        echo "No scale systematics found!"
+    fi
     
     echo "Done."
 
     popd
 done
 
 mkdir -p $1/slurm/output
-if [[ ${directories[*]} =~ $1_for_signal ]]; then echo "Moving signal files to main folder..."; mv $1_for_signal/slurm/output/*.root $1/slurm/output ; fi
-if [[ ${directories[*]} =~ $1_for_data ]]; then echo "Moving data files to main folder..."; mv $1_for_data/slurm/output/*.root $1/slurm/output ; fi
-if [[ ${directories[*]} =~ $1_for_MCbkg ]]; then echo "Moving MC bkg files to main folder..."; mv $1_for_MCbkg/slurm/output/*.root $1/slurm/output ; fi
+if [[ ${directories[*]} =~ $1_for_signal ]]; then echo "Moving signal files to main folder..."; cp $1_for_signal/slurm/output/*.root $1/slurm/output ; fi
+if [[ ${directories[*]} =~ $1_for_data ]]; then echo "Moving data files to main folder..."; cp $1_for_data/slurm/output/*.root $1/slurm/output ; fi
+if [[ ${directories[*]} =~ $1_for_MCbkg ]]; then echo "Moving MC bkg files to main folder..."; cp $1_for_MCbkg/slurm/output/*.root $1/slurm/output ; fi
```

Comment:
why `cp` instead of `rm` ?

Reply:
It's a temporary thing, I need a one-to-one correspondence between the root files and the error log files to check what's going on... once these jobs don't fail anymore (it's a problem of the cluster - not of the code - so the PR is safe) I will change it again

Reply:
ok then :+1: !